main.js

```diff
@@ -1,10 +1,66 @@
 window.onload = () => {
-    let menuContainer = document.getElementById("mob-container");
-    let menu = document.querySelectorAll(".mob-click");
-    menu.forEach(el => {
-        el.addEventListener('click', () => {
-            menuContainer.classList.toggle('disapear');
-        })
+  const menuContainer = document.getElementById('mob-container');
+  const menu = document.querySelectorAll('.mob-click');
+  menu.forEach((el) => {
+    el.addEventListener('click', () => {
+      menuContainer.classList.toggle('disapear');
+    });
+  });
 
-    })
-}
+  const speakersList = [
```

Comment:
The section "Featured speakers" is generated dynamically with JavaScript 🔥

Reply:
### OPTIONAL 

You could change a bit of info in your speakers so that is obvious to the eye that you are using dynamic objects when creating your section 😎